pandas/core/arrays/base.py

```diff
@@ -1149,6 +1149,7 @@ def take(
         *,
         allow_fill: bool = False,
         fill_value: Any = None,
+        axis: AxisInt = 0,
```

Comment:
i dont mind having this keyword, but we shouldn't silently ignore it. i.e. needs validation.  and im guessing you intentionally ignore it in tile_compat?

Reply:
it's used in
```
return cast(AnyArrayLikeT, arr.take(taker, axis=axis))
```
(last line of `tile_compat`)

Reply:
I think there's functions where passing anything other than `axis=0` errors, I'll see if that can be done here